src/recursive_sorting/recursive_sorting.py

```diff
@@ -3,15 +3,46 @@ def merge( arrA, arrB ):
     elements = len( arrA ) + len( arrB )
     merged_arr = [0] * elements
     # TO-DO
+    new_arr = []
+    
+    for i in range(0, elements):
+        if len(arrA) == 0:
+            new_arr.append(arrB[0])
+            arrB.pop(0)
+            continue
+        if len(arrB) == 0:
+            new_arr.append(arrA[0])
+            arrA.pop(0)
+            continue
+        
+        if arrA[0] < arrB[0]:
+            new_arr.append(arrA[0])
+            arrA.pop(0)
+        else:
+            new_arr.append(arrB[0])
+            arrB.pop(0) 
+
+    return new_arr
+
+print(merge([1, 3, 5], [2, 4]))
+
     
-    return merged_arr
 
 
 # TO-DO: implement the Merge Sort function below USING RECURSION
 def merge_sort( arr ):
     # TO-DO
 
-    return arr
+    if len(arr) <= 1:
+        return arr
+    middle = len(arr) // 2
+    left = merge_sort(arr[:middle])
+    right = merge_sort(arr[middle:])
+    
+
+    return merge(left, right)
+
```

Comment:
Nice job getting the recursion working!  The function is so small but it ends up doing so much...

Imagine writing this without recursion, the function would end up looking quite a bit more complicated!